caver/patcher.py

```diff
@@ -41,7 +41,7 @@ def patch_files(patch_data: dict, output_dir: Path, progress_update: Callable[[s
 
     i += 1
     progress_update("Copying MyChar...", i/total)
-    patch_mychar(patch_data["mychar"], output_dir)
+    patch_mychar(patch_data["mychar"], output_dir, patch_data["platform"] == "tweaked")
```

Comment:
then instead of this expression, you can make 2x mychar a property on the enum itself